cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -137,45 +138,49 @@ export class Assessment extends Application {
     }
 
     protected selectAnswers(risk: string): void {
-        for (let i = 0; i < 5; i++) {
-            cy.get(questionBlock).each(($question) => {
-                let totalOptions = $question.find(stack).children("div").length;
-                let optionToSelect: number;
-                if (risk === "low") {
-                    optionToSelect = totalOptions - 1;
-                    this.clickRadioOption($question, optionToSelect);
-                } else if (risk === "medium") {
-                    cy.wrap($question)
-                        .children()
-                        .find("div.pf-v5-l-split__item")
-                        .then(($questionLine) => {
-                            /* These 3 questions generate high risk with mean options, 
-                            hence to keep risk to medium, select last options for these set of specific questions */
-                            if (
-                                $questionLine.text() ===
-                                    "Does the application have legal and/or licensing requirements?" ||
-                                $questionLine.text() ===
-                                    "Does the application require specific hardware?" ||
-                                $questionLine.text() ===
-                                    "How is the application clustering managed?"
-                            ) {
-                                optionToSelect = totalOptions - 1;
-                            } else {
-                                optionToSelect = Math.floor(totalOptions / 2);
-                            }
+        cy.get(assessmentBlock)
+            .its("length")
+            .then((count) => {
+                for (let i = 0; i < count - 1; i++) {
+                    cy.get(questionBlock).each(($question) => {
+                        let totalOptions = $question.find(stack).children("div").length;
+                        let optionToSelect: number;
+                        if (risk === "low") {
+                            optionToSelect = totalOptions - 1;
                             this.clickRadioOption($question, optionToSelect);
-                        });
-                } else {
-                    optionToSelect = 1;
-                    this.clickRadioOption($question, optionToSelect);
+                        } else if (risk === "medium") {
```

Comment:
One last consideration in case you have the bandwidth:

We're using hardcoded strings in lot of places and that can be dangerous.

If someone tries to call the method using "Medium" instead of "medium" the test will fail and it may be hard to find the root cause.

A good idea to solve this issue is to create an enum (`AssessmentRisks`) and change the method signature so it receives an `AssessmentRisk` instead of a string, with this, there is no way someone missess the correct value.

Reply:
https://issues.redhat.com/browse/MTA-1737